src/pipeline.py

```diff
@@ -1,34 +1,29 @@
 """Pipeline used to create a stable diffusion dataset from a set of initial prompts."""
 import logging
-import fsspec
 from pathlib import Path
 
 from fondant.pipeline import ComponentOp, Pipeline
 
 logger = logging.getLogger(__name__)
 
-# General configs
-HF_USER = None  # Insert your huggingface username here
-HF_TOKEN = None  # Insert your HuggingFace token here
 BASE_PATH = "./data_dir"
-N_ROWS_TO_LOAD = 10  # Set to None to load all rows
-
-# Create data directory if it doesn't exist and if it's a local path
-if fsspec.core.url_to_fs(BASE_PATH)[0].protocol == ("file", "local"):
```

Comment:
we only want to create a path if the base path is local. 

Maybe we can create it with fsspec but then we assume that the user has access to the base path

```python
fs = fsspec.core.url_to_fs(BASE_PATH)
fs.mkdir(BASE_PATH)
```

Reply:
I would assume a local path for the notebook and keep the code simple.